acstools/acsphotcte.py

```diff
@@ -200,21 +200,18 @@ def _check_inputs(self, **inputs):
                         return key
         # the last step is to ensure all the iterable inputs are the same length
         nl = "\n"
-        try:
-            # First we check if there were any iterable inputs
-            if iterable_lengths:
-                # if there are we make sure theyre all the same size
-                assert len(set(iterable_lengths)) == 1
-        except AssertionError:
+        # First we check if there were any iterable inputs.
+        # If there are, we make sure theyre all the same size.
+        if iterable_lengths and len(set(iterable_lengths)) != 1:
```

Comment:
@nmiles2718 , this should not change how your code works. I needed to fix the logic to comply with `bandit` security audit. FYI.

https://docs.openstack.org/bandit/latest/plugins/b101_assert_used.html